Dockerfile

```diff
@@ -72,13 +72,13 @@ RUN set -ex && cd ~ \
   && : Install Node 10.x \
   && curl -sS https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
   && echo "deb https://deb.nodesource.com/node_10.x $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/nodesource.list \
-  && apt-get -qq update \
```

Comment:
@chrisgilmerproj Why did these `update` commands get removed? It used to be the case that `apt-get` wouldn't see the packages from the newly added apt source unless that got ran.

Reply:
I think I see the problem, let me make a change to our dockerfile and push a new update.

Reply:
See #84 for updates